### PR TITLE
Plaintext Task Management

### PR DESCRIPTION
**Pull request by jpknwls**

### CODE REVIEW

**Comment by jpknwls** [comment]
Also included are some updates to the front matter of the latest `Growing Knowledge Bases` blog post.

File: vault/blog.2022.2022-04-05-growing-knowledge-bases-from-the-bottom-up.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
 ---
 id: 072ed07tikrhv6e4ilwcv2q
 title: Growing Knowledge Bases from the Bottom Up
-desc: ''
-updated: 1649124319769
+desc: >-
```
Comment (kevinslin) [inline vault/blog.2022.2022-04-05-growing-knowledge-bases-from-the-bottom-up.md:4]: what happened here? this is invalid yml

File: vault/blog.2022.2022-04-19-plaintext-task-management-vscode.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
+---
+id: 1jynrsx70fh3mihhrhikrf8
+title: Plaintext Task Management with VSCode
```
Comment (kevinslin) [inline vault/blog.2022.2022-04-19-plaintext-task-management-vscode.md:3]: i reconsidered, lets change the title back to `with Dendron and VSCode`

**Review by kevinslin — CHANGES_REQUESTED** [review]
two comments, otherwise good

File: vault/blog.2022.2022-04-05-growing-knowledge-bases-from-the-bottom-up.md (hunk 1 above)
Comment (jpknwls) [reply vault/blog.2022.2022-04-05-growing-knowledge-bases-from-the-bottom-up.md:4]: not sure, fixed

**Pull request merged**